platform/linux-generic/odp_timer.c

```diff
@@ -217,7 +218,7 @@ typedef struct timer_global_t {
 	/* Multiple locks per cache line! */
 	_odp_atomic_flag_t ODP_ALIGNED_CACHE locks[NUM_LOCKS];
 #endif
-
+	odp_bool_t use_inline_timers;
```

Comment:
Why do we need two separate variables: one here and one in `odp_global_rw`?

Reply:
The value of `odp_global_rw.inline_timers` changes depending on if timer pools have been created to minimize overhead when timers are not actually used.

`timer_global_t.use_inline_timers` is set once during global init based on the config value. This avoids having to read the config file in `timer_pool_new()`.